### ISSUE TITLE
Failed to train the model by following the instructions given with the command "th train.lua"

### ISSUE THREAD

**Issue by sami5596** [facebookresearch/deepmask#122]
I'm trying to train the model by following the already given instructions and using the dataset as mentioned in the instructions i.e. coco 2014 dataset. But whenever I try to run command "th train.lua" I get the following error: 

shoaib@shoaib-HP-ENVY-Notebook:~/deepmask$ th train.lua 
-- ignore option rundir	
-- ignore option dm	
-- ignore option reload	
-- ignore option gpu	
-- ignore option datadir	
| running in directory /home/shoaib/deepmask/exps/deepmask/exp	
| number of paramaters trunk: 15198016	
| number of paramaters mask branch: 1608768	
| number of paramaters score branch: 526337	
| number of paramaters total: 17333121	
convert: data//annotations/instances_train2014.json --> .t7 [please be patient]
/home/shoaib/torch/install/bin/luajit: ...e/shoaib/torch/install/share/lua/5.1/threads/threads.lua:183: [thread 2 callback] /home/shoaib/torch/install/share/lua/5.1/coco/CocoApi.lua:142: Expected value but found T_END at character 1
stack traceback:
	[C]: in function 'decode'
	/home/shoaib/torch/install/share/lua/5.1/coco/CocoApi.lua:142: in function '__convert'
	/home/shoaib/torch/install/share/lua/5.1/coco/CocoApi.lua:128: in function '__init'
	/home/shoaib/torch/install/share/lua/5.1/torch/init.lua:91: in function </home/shoaib/torch/install/share/lua/5.1/torch/init.lua:87>
	[C]: in function 'CocoApi'
	/home/shoaib/deepmask/DataSampler.lua:25: in function '__init'
	/home/shoaib/torch/install/share/lua/5.1/torch/init.lua:91: in function </home/shoaib/torch/install/share/lua/5.1/torch/init.lua:87>
	[C]: in function 'DataSampler'
	/home/shoaib/deepmask/DataLoader.lua:36: in function </home/shoaib/deepmask/DataLoader.lua:30>
	[C]: in function 'xpcall'
	...e/shoaib/torch/install/share/lua/5.1/threads/threads.lua:234: in function 'callback'
	/home/shoaib/torch/install/share/lua/5.1/threads/queue.lua:65: in function </home/shoaib/torch/install/share/lua/5.1/threads/queue.lua:41>
	[C]: in function 'pcall'
	/home/shoaib/torch/install/share/lua/5.1/threads/queue.lua:40: in function 'dojob'
	[string "  local Queue = require 'threads.queue'..."]:13: in main chunk
stack traceback:
	[C]: in function 'error'
	...e/shoaib/torch/install/share/lua/5.1/threads/threads.lua:183: in function 'dojob'
	...e/shoaib/torch/install/share/lua/5.1/threads/threads.lua:264: in function 'synchronize'
	...e/shoaib/torch/install/share/lua/5.1/threads/threads.lua:142: in function 'specific'
	...e/shoaib/torch/install/share/lua/5.1/threads/threads.lua:125: in function 'Threads'
	/home/shoaib/deepmask/DataLoader.lua:40: in function '__init'
	/home/shoaib/torch/install/share/lua/5.1/torch/init.lua:91: in function </home/shoaib/torch/install/share/lua/5.1/torch/init.lua:87>
	[C]: in function 'DataLoader'
	/home/shoaib/deepmask/DataLoader.lua:21: in function 'create'
	train.lua:101: in main chunk
	[C]: in function 'dofile'
	...oaib/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x00405d50
convert: data//annotations/instances_train2014.json --> .t7 [please be patient]

Does anybody has any idea how to resolve this issue? Kindly guide me through.

**Comment by a651529341** [comment]
我用TORCH_LUA_VERSION=LUA52 ./install.sh 安装torch7解决的
